src/App.tsx

```diff
@@ -42,7 +43,7 @@ const App: React.FC = () => {
               </Switch>
             </div>
           </Content>
-          <Footer />
+          {/*<Footer />*/}
```

Comment:
Might as well keep this or no?

Reply:
Designs didn't have a footer but doesn't hurt to have for now

Reply:
We'll probably throw it out at some point so doesn't really matter if you remove or leave it